inc/saiport.h

```diff
@@ -2528,6 +2528,15 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_JSON_FORMATTED_DEBUG_DATA_SIZE,
 
+    /**
+     * @brief Unreliable Loss of Signal
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_UNRELIABLE_LOSS_OF_SIGNAL,
```

Comment:
@JaiOCP Can we also consider changing it to SAI_PORT_ATTR_UNRELIABLE_LOS instead?